src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionHelpers.cs

```diff
@@ -251,8 +256,11 @@ private static bool IsCodeActionNotSupportedByLSP(IUnifiedSuggestedAction sugges
             CodeActionOptionsProvider fallbackOptions,
             ICodeFixService codeFixService,
             ICodeRefactoringService codeRefactoringService,
+            bool hasVsLspCapability,
             CancellationToken cancellationToken)
         {
+            // Filter the configure and suppress fixer if it is not VS LSP, because it would generate many nested code actions.
+            // Tracking issue: https://github.com/microsoft/language-server-protocol/issues/994 
```

Comment:
are the changes in this method needed?

Reply:
No, I am just going to push a clean up commit